extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -155,6 +159,16 @@ export function getElement(selector, el, selectionMethod) {
   if (!el) {
     return null;
   }
+
+  // Special case for root selector.
+  if (selector == ':host' || selector == ':root') {
```

Comment:
I would prefer to do this in another PR. Currently, if there is no `visibilitySpec`, this method is never called. If we want to default selector to something, instrumentation.js will need to be updated to either call this always or add the selector there for analytics inside embeds.